webpage readme.md中文版部署文档

Jonyzqw opened the pull request:
readme.md 1.0

Jonyzqw committed:
Update README.md

webpage 中文版部署文档

Signed-off-by: Jonyzqw <137276052+Jonyzqw@users.noreply.github.com>

Jonyzqw committed:
Create readme.md

Signed-off-by: Jonyzqw <a2467449838@163.com>

Jonyzqw committed:
Add files via upload

Signed-off-by: Jonyzqw <a2467449838@163.com>

Jonyzqw committed:
Update README.md

Signed-off-by: Jonyzqw <a2467449838@163.com>

Jonyzqw committed:
Create readme.md

Signed-off-by: Jonyzqw <a2467449838@163.com>

Jonyzqw committed:
Add files via upload

Signed-off-by: Jonyzqw <a2467449838@163.com>

Jonyzqw committed:
Update mnist.md

Signed-off-by: Jonyzqw <a2467449838@163.com>

Jonyzqw committed:
Update README.md

Signed-off-by: Jonyzqw <a2467449838@163.com>

Jonyzqw commented:
wenbpage的中文部署文档和mnist demo的运行文档 的readme.cd都已pr给您了

whybeyoung commented on docs/加载器/examples/mnist.md:
```diff
@@ -1,280 +1,147 @@
+# 准备环境
+python 版本请选用 3.9+（[也可以下载我们的docker镜像](https://github.com/iflytek/aiges/releases "也可以下载我们的docker镜像"))
+# 1. 首先，在docker中配置容器环境（无需gpu），后续demo的运行也在容器中。
+    docker run -itd --name mnist2 -p 1889:1888 iflyopensource/aiges-gpu:10.1-1.17-3.9.13-ubuntu1804-v3.3.7 bash
```
这里版本号都不要写死，最好统一换成一个命令获取的方式，获取最新版本号，这里用变量替代

Jonyzqw replied:
好的

whybeyoung commented on docs/加载器/examples/mnist.md:
```diff
@@ -1,280 +1,147 @@
+        └── wrapper.py
+
+## 4.3. 解压加载器
+    tar zxvf aiges_3.3.7_linux_amd64.tar.gz -C mnist
```
版本号问题同上

Jonyzqw replied:
好的

Jonyzqw committed:
Update mnist.md

将docker run -itd --name mnist2 -p 1889:1888 iflyopensource/aiges-gpu:10.1-1.17-3.9.13-ubuntu1804-v3.3.7 bash和tar zxvf aiges_3.3.7_linux_amd64.tar.gz -C mnist的版本号都设置为获取最新版本访问方式

Signed-off-by: Jonyzqw <a2467449838@163.com>

Jonyzqw committed:
Update mnist.md

Signed-off-by: Jonyzqw <a2467449838@163.com>

whybeyoung commented:
https://github.com/whybeyoung
